interfaces/Glitter/templates/static/javascripts/knockout-extensions.js

```diff
@@ -51,6 +51,10 @@ ko.bindingHandlers.longText = {
             // Nothing special
             outputText += value.join('<br />');
         }
+
+        // Replace link to our website
+        outputText = outputText.replace('https://sabnzbd.org/not-complete', '<a href="https://sabnzbd.org/not-complete" class="history-status-dmca" target="_blank">https://sabnzbd.org/not-complete</a>')
+
```

Comment:
You have the same URL in three different places.
It would be better to put it in "constants.py" and send it as a parameter to the Web-UI.